docker/pandoc-lambda/function/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile
```

Comment:
Hmm weird, how did we manage to run this with different versions of pip/pip-tools...... Oh well.